Week2/homework/map-filter.js

```diff
@@ -2,11 +2,12 @@
 
 function doubleOddNumbers(numbers) {
   // Replace this comment and the next line with your code
-  console.log(numbers);
+  const newNumber = numbers.filter(number => (number % 2 === 1 ? number : false)).map(x => x + x);
```

Comment:
The anonymous (arrow) function that you pass as an argument to the `filter` method should return a boolean (i.e. `true` or `false`. In your case you are return `false` or the number itself. It still works because zero is 'falsy' and non-zero is 'truthy', but that is just sheer luck. Can you fix this by letting the function return a genuine boolean at all times?